Sources/String+Sweetness.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public extension String {
+    public var isValidEmail: Bool {
+        let emailRegEx = ".*@.*"
```

Comment:
I would like if we had this here anyway. Something that checks

[validcharacters]@[validcharacters].[validcharacters]

`validcharacters` of course won't contain @ or .